web/src/api/request.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  createAxle,
+  requestHeadersInterceptor,
+  type RunnerMethod,
+  type AxleRequestConfig
+} from '@varlet/axle'
+import { createUseAxle } from '@varlet/axle/use'
+import type { Options } from './types'
+
+export const axle = createAxle({
+  baseURL: '/api'
+})
+
+axle.useRequestInterceptor(
+  requestHeadersInterceptor({
+    headers: () => ({
+      Authorization: ''
+    })
+  })
+)
+
+axle.useResponseInterceptor({
+  onFulfilled(response) {
+    const { status, statusText } = response.data
+    if (status !== 200 && statusText) {
```

Comment:
Does this need to log the status?

Reply:
Maybe there is a misunderstanding of the word, this should be the returned message